Assets/GothicVR/Scripts/UI/FontManager.cs

```diff
@@ -0,0 +1,34 @@
+using GVR.Phoenix.Interface;
+using GVR.Util;
+using System.Collections.Generic;
+using TMPro;
+
+namespace GVR.Manager
+{
+    public class FontManager : SingletonBehaviour<FontManager>
+    {
+        public Dictionary<string, TMP_FontAsset> fontDictionary = new Dictionary<string, TMP_FontAsset>(); // Dictionary to store font assets for each tag
```

Comment:
Please don't use a dictionary. Justification:
* We only have 2 elements. And it won't change.
* We need to have additional keys named ["MenuUI"]/["Subtitle"]
* If we have only a handful of entries it's easier to see directly which ones they are.

I recommend:

```suggestion
        public TMP_FontAsset fontSubtitle;
        public TMP_FontAsset fontMenuUI;
```

Reply:
Tbh I would leave it as it is. The only reason why I would really go for a change would be performance.
It's obvious what it does. And we might wanna use more fonts in the future. So that's a good foundation.

Reply:
I see your point when looking down the foreach loop. This is a valid use case as we would otherwise need to add everything manually without a loop!

Now let's be super cool and use our newly created ConstantManager to reference the names. This is where it comes in handy:

```c#
public Dictionary<string, TMP_FontAsset> fontDictionary = new (); // Dictionary to store font assets for each tag

public void Create()
{
    fontDictionary[ConstantsManager.I.MenuFontTag] = GameData.I.GothicMenuFont;
    fontDictionary[ConstantsManager.I.SubtitleFontTag] = GameData.I.GothicSubtitleFont;
}
```

Reply:
Followed your suggestions. Done.